### ISSUE TITLE
Accessibility Check Failures (table regularity and list tags) with tagged pdf

### ISSUE THREAD

**Issue by henningpohl** [borisveytsman/acmart#614]
I tested tagged pdf output with a recent paper of mine and got some errors in the accessibility report generated by acrobat. Here's a reduced example file:
```latex
\DocumentMetadata{
  lang        = en,
  pdfstandard = ua-2,
  pdfstandard = a-4f, %or a-4
  tagging=on,
  tagging-setup={math/setup=mathml-SE} 
}
\documentclass[sigconf]{acmart}

\usepackage{booktabs}

\title{Some Title}
\author{Some Author}
\email{author@acm.org}

\begin{document}

\maketitle

\section{Footnote Test}

Some text\footnote{With a footnote}. 
And some more text.


\section{Table Test}

\begin{table}[h]
\caption{Some caption}
\begin{tabular}{llrrrl}
\toprule
  Question & Factor & df & $\chi^2$ & \multicolumn{2}{c}{p-value} \\ 
\midrule
  Q1 & F1 &    2 & 1.0 & 0.1 &   \\ 
  Q1 & F2 &    2 & 1.0 & <0.001 & *** \\ 
  Q1 & F1:F2 & 4 & 1.0 & 0.5 &  \\
\bottomrule
\end{tabular}
\end{table}

\end{document}
````

The errors are the following:

<img width="240" height="353" alt="Image" src="https://github.com/user-attachments/assets/a9bf25d9-c0c7-4246-9542-2a12e4f70251" />

For the table, this is probably due to the use of multicolumn, which resulted in the mismatched number of table cells per row:
<img width="187" height="441" alt="Image" src="https://github.com/user-attachments/assets/b52d6a5a-65ff-4924-b8e2-7a430afefbed" />

It's less clear to me what the issue with the footnote is:
<img width="1907" height="391" alt="Image" src="https://github.com/user-attachments/assets/64fff90d-8706-4310-a6ff-dd23956a33bf" />

The complaint from Acrobat (https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html?trackingid=KACNN#LblLBody) relates to child elements of lists, but there is no list here. So maybe this is due to the <lbl> tag used outside? 

Both these are not really things for acmart to fix. I know tables already might need some manual markup (e.g., https://latex3.github.io/tagging-project/documentation/usage-instructions#handling-tables-tabular-in-the-document), but not sure about multicolumns.

**Comment by senthilmm** [comment]
Adding the following text resolves the Acrobat accessibility check error for the footnotes.

\IfDocumentMetadataT{%
\AssignStructureRole{fnote/mark}{Reference}}

**Comment by henningpohl** [comment]
Thanks! And interesting, wondering how that's not the default then. I suppose this should then be added to acmart?

As for the multicolumns, I found that those are actually supposed to work out of the box (see https://texdoc.org/serve/latex-lab-table/0#page=3.57). But that did not seem to have worked here.

**Comment by henningpohl** [comment]
Can't seem to get `\AssignStructureRole` to work though. 

<img width="1005" height="279" alt="Image" src="https://github.com/user-attachments/assets/7d1ff899-e4af-46cf-9b7d-f9a0403d5408" />

**Comment by henningpohl** [comment]
Never mind, somehow that project used an older lualatex compiler. Works with latest

**Comment by henningpohl** [comment]
The colspan issue might be due to how latex does that annotation. That cell does have an attribute class with that setting:

<img width="467" height="366" alt="Image" src="https://github.com/user-attachments/assets/7383742c-defb-49e7-ad03-f7c269a765c4" />

But the table cell does not have that set:

<img width="538" height="530" alt="Image" src="https://github.com/user-attachments/assets/d5ad5247-68a7-4bb2-a670-cbcf94c76698" />

So this might just be the acrobat validator not working with what latex generates. However, that's also the validator I've seen publication chairs use so far, so that might need to be communicated to them.

**Comment by Carminiumn** [comment]
The list tag flag is a **false positive** from Acrobat's accessibility checker.

It assumes every label tag (`<Lbl>`) has to be paired with a list item body (`<LBody>`), as if labels can only be used in lists. But that's not what the PDF spec says—`<Lbl>`can tag a range of content, and footnote markers are given as an example of the tag's use outside of lists.

Someone [flagged the same issue on Adobe community back in 2021](https://community.adobe.com/questions-9/accessibility-lbl-lbody-failed-1262695) (the top reply said it's standard-compliant but that ATs wouldn't handle it well. _That is no longer the case_ as we recently tested it with NVDA and had no issues). Kind of wild that it's been five years and Adobe still hasn't fixed this.

ISO 32000-2:2020 Table 368 Row 1:

> Structure Type: Lbl
> ...
> The following are just a few of the possible types of content that could be marked as Lbl:  
> - bullets for list items in an unordered list 
> - digits or characters used for numbering list items in an ordered list 
> - the number in the Caption for a Figure or Table 
> - in a form, the label of a form field 
> - **in footnotes, the number or symbol matching the reference from the text**  
> - in headings, the enumeration of the headings

And Figure H.6 gives an example tag use:

<img width="464" height="440" alt="Example from the PDF standard showing the structure of a table of contents. A TOC entry, “1. Chapter One … 3,” is divided into marked-content elements: <Lbl> for the entry number “1.,” <Reference> for “Chapter One,” <NonStruct> for the dot leaders, and another <Reference> for the page number “3.” The <Lbl> element is used on its own, without an <LBody> element." src="https://github.com/user-attachments/assets/9acbc1b6-c20a-4041-baa9-b48a622e2a60" />

See the arrow pointed Table of Contents entry ("Chapter 1"), which is tagged with `<Lbl>` for the entry number, `<Reference>` for the title and page number, and `<NonStruct>` for dot leaders. It's basically `<Lbl>` used without an accompanying `<LBody>` element.
